gordo/machine/dataset/filter_rows.py

```diff
@@ -133,8 +134,15 @@ def pandas_filter_rows(df, filter_str: str, buffer_size: int = 0):
     # and returns a pd.Series
     else:
         pandas_filter = df.eval(filter_str)
+
     apply_buffer(pandas_filter, buffer_size=buffer_size)
-    return df[list(pandas_filter)]
+    df = df[list(pandas_filter)]
+    logger.info("Shape of data after numerical filtering: %s", df.shape)
+    logger.info("Applying global min/max filtering")
+    mask = ((df > -1000) & (df < 50000)).all(1)
```

Comment:
I do not think this is a good place to put these values here. It could have these values by default but the user should be able to change these values as `min_global_threshold` and `max_global_threshold` in the gordo resource config.

Reply:
Also, can we double-check these values?

Reply:
Sure! I agree. This was obviously a "quick fix", and to show what is wanted.
It is however not completely clear to me how those parameters would be picked up from the config and where they eventually would be evaluated.
Any advice would be appreciated.

Reply:
I agree with @koropets that these default values should be set to function args defaults with the possibility to customize them from the config, same way as we are passing the `row_filter_buffer` for example.
@henrikny I also have no problem with this filtering being located here.
@hjarraya these were the values discussed yesterday with the SME's and seem to be reasonable defaults from their perspective.